website/blog/2021-12-18_DAW_Adventures_IN_WDIO.md

```diff
@@ -0,0 +1,82 @@
+
+# Adventures in WDIO with Paul Grossman the DarkArtsWizard 
+http://github.com/darkartswizard
+
```

Comment:
Can you prefix the file similar to other blog posts, e.g.:
```
---
title: WebdriverIO, meet Serenity/JS
author: Jan Molak
authorURL: https://www.linkedin.com/in/janmolak/
authorImageURL: https://avatars.githubusercontent.com/u/1089173?s=400&v=4
---
```

Reply:
Changes made as requested